src/django_simple_nav/nav.py

```diff
@@ -141,8 +141,10 @@ def get_active(self, request: HttpRequest) -> bool:
         parsed_request = urlparse(request.build_absolute_uri())
 
         if (
-            parsed_url.scheme != parsed_request.scheme
-            or parsed_url.netloc != parsed_request.netloc
+            parsed_url.scheme
+            and (parsed_url.scheme != parsed_request.scheme)
+            or parsed_url.netloc
+            and (parsed_url.netloc != parsed_request.netloc)
```

Comment:
Please group these by the order of operations so that it's easier to maintainer. 

I think you were going for roughly: 

```python
if (
    (parsed_url.scheme and parsed_url.scheme != parsed_request.scheme)
    or
    (parsed_url.netloc and parsed_url.netloc != parsed_request.netloc)
):
```

Reply:
Boolean operators have higher precedence than comparison operators, so wouldn't `(parsed_url.scheme and parsed_url.scheme != parsed_request.scheme)` be the same as `((parsed_url.scheme and parsed_url.scheme) != parsed_request.scheme)`? This would compare a boolean to a string, which will always fail. What I wanted to express was `(parsed_url.scheme and (parsed_url.scheme != parsed_request.scheme))`.

Reply:
OK, I have added parentheses around the operators of the `or`.